config/default.schema.json

```diff
@@ -532,6 +532,11 @@
 			"default": false,
 			"description": "Enables the legacy Hydra endpoints."
 		},
+		"FEATURE_SHOW_ALERTS_ON_HOMEPAGE": {
```

Comment:
name should be like FEATURE_ALERTS_ON_HOMEPAGE_ENABLED
the ending  "enabled", is a convention for feature flags
FEATURE_ALERTS_ENABLED is already an existing flag, pls try to group them

Reply:
done